### PR TITLE
task-2frontendupdated

### PR DESCRIPTION
**Pull request by kavyasri189**

### CODE REVIEW

File: slr/basic_app/templates/basic_app/frontend_submission.html [hunk 1]
```diff
@@ -2,10 +2,84 @@
...
+        <p><mark>Lorem ipsum</mark> dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur.<strong> Quis aute </strong> reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+    </section>
+    <section class="child">
+        <img style="float: centre" ; padding: 15px 25px 10px 20px; src="C:\Users\KAVYASRI DACHARLA\WSoC-TestRepo1\slr\static\basic_app\doggo-bread.jpg" alt="Flowers in Chania" width="560" height="345">
```
Comment (dat-adi) [inline slr/basic_app/templates/basic_app/frontend_submission.html:71]: The path to the image is preferably relative whenever you are working with linking images, and the such.
Look into [this](https://www.w3schools.com/html/html_filepaths.asp) to understand how it works, and the potential benefits of working with relative file paths.

**Review by dat-adi — COMMENTED** [review]
## Review
 - [ ] Followed branch creation instructions. 
 - [x] Accomplished all the front end tasks requested.
 - [ ] Accomplished all the back end tasks requested.
 - [ ] Pushed a commit to the PR for every change made.
 - [x] Has bread doggo's permission.

File: slr/basic_app/templates/basic_app/frontend_submission.html (hunk 1 above)
Comment (kavyasri189) [reply slr/basic_app/templates/basic_app/frontend_submission.html:71]: Yes that's true relative path is better.Its my mistake

Comment (dat-adi) [reply slr/basic_app/templates/basic_app/frontend_submission.html:71]: Don't worry about it. I'm just pointing out better ways to solve issues.
Try to work with it, the next time you pick up a project. :)

Comment (kavyasri189) [reply slr/basic_app/templates/basic_app/frontend_submission.html:71]: yeah sure.Thank you

**Pull request closed without merging**